x/evm/watcher/types.go

```diff
@@ -449,19 +460,19 @@ type TransactionReceipt struct {
 	ContractAddress       *common.Address `json:"contractAddress"`
 	GasUsed               hexutil.Uint64  `json:"gasUsed"`
 	originBlockHash       common.Hash
-	BlockHash             string         `json:"blockHash"`
-	BlockNumber           hexutil.Uint64 `json:"blockNumber"`
-	TransactionIndex      hexutil.Uint64 `json:"transactionIndex"`
-	tx                    *types.MsgEthereumTx
-	From                  string          `json:"from"`
-	To                    *common.Address `json:"to"`
+	BlockHash             string               `json:"blockHash"`
+	BlockNumber           hexutil.Uint64       `json:"blockNumber"`
+	TransactionIndex      hexutil.Uint64       `json:"transactionIndex"`
+	Tx                    *types.MsgEthereumTx `json:"tx"`
```

Comment:
I will break eth_getTransactionReceipt method, because TransactionReceipt is used by eth_getTransactionReceipt.